File: SecondYear/Parth27904_ParthAgarwal_2226CSE(AI&ML)1015_2/Week_2/armstrong.java
import java.util.Scanner;

public class armstrong {
	
	public static void main(String[] args) {
		// Write your code here
		Scanner scanner = new Scanner(System.in);
		int n = scanner.nextInt();
		int count = 0;
		int temp = n;
		int temp2 = n;
		int digit = 0;
		while(temp!=0){
			digit = temp%10;
			count++;
			temp = temp/10;
		}
		int sum = 0;
		while(temp2!=0){
			digit = temp2%10;
			sum += (int)Math.pow(digit, count);
			temp2 = temp2/10;
		}
		if(sum==n){
			System.out.println("true");
		}
		else{
			System.out.println("false");
		}
        scanner.close();
	}
}

File: SecondYear/Adityacheeze_AdityaMohanGupta_2226CSE(AI&ML)1018_2/Week_2/Armstrong.cpp
bool checkArmstrong(int n){
	//Write your code here
	int count = 0;
	int temp = n;
	while(temp != 0){
		temp = temp / 10;
		count++;
	}
	temp = n;
	int sum = 0;
	while(temp != 0){
		int digit = temp % 10;
		sum = sum + pow(digit,count);
		temp = temp /10;
	}
	if(sum == n){
		return true;
	}
	return false;
}
